.changeset/chilly-deers-count.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux-private/preview-middleware-client': minor
```

Comment:
Here we are missing a @sap-ux/preview-middleware where it is consumed.

cc @voicis